.github/actions/setup-node/action.yml

```diff
@@ -66,6 +71,7 @@ runs:
     - name: Set up Node.js
       uses: actions/setup-node@a0853c24544627f65ddf259abe73b1d18a591444 # v5.0.0
       with:
+        always-auth: ${{ inputs.always-auth }}
```

Comment:
Shouldn't you give it a description and default value like for the `package-manager-cache`?